index.js

```diff
@@ -73,6 +73,7 @@ function setupSchema (schema, policy, all, cache, skip, storage, onHit, onMiss,
       }
     }
   }
+  if (queryKeys.length) { throw new Error('Query does not match schema') }
```

Comment:
It would be useful for the user to know the wrong configuration to fix it faster

```suggestion
  if (queryKeys.length) { throw new Error('Query does not match schema: ${queryKeys}') }
```